snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedDocument.java

```diff
@@ -131,10 +132,10 @@ public B effectiveTime(final long effectiveTime) {
 	}
 	
 	public static class Fields extends RevisionDocument.Fields {
-		public static final String MODULE_ID = "moduleId";
-		public static final String RELEASED = "released";
-		public static final String ACTIVE = "active";
-		public static final String EFFECTIVE_TIME = "effectiveTime";
+		public static final String MODULE_ID = SnomedComponent.Fields.MODULE_ID;
```

Comment:
This introduces coupling between the domain and the indexed document world, which makes future changes to either of the classes harder (either one has to remember to rename properties in both classes, or rename in one and "disconnect" the constant value in the other)

Reply:
Then let's connect both to `SnomedRf2Headers` constants, where applicable.